configs/retinanet/retinanet_r50_fpn_1x_coco.yml

```diff
@@ -0,0 +1,10 @@
+_BASE_: [
+  '../datasets/coco_detection.yml',
+  '../runtime.yml',
+  '_base_/retinanet_r50_fpn.yml',
+  '_base_/optimizer_1x.yml',
+  '_base_/retinanet_reader.yml'
+]
```

Comment:
Please add a README and modelzoo.

Reply:
OK, how to upload files to https://paddledet.bj.bcebos.com/... ?

Reply:
> OK, how to upload files to https://paddledet.bj.bcebos.com/... ?

You can upload on aistudio so that we can download, or just send to nifeng03@baidu.com. Thank you very much.